playerbot/PlayerbotAI.h

```diff
@@ -568,6 +579,8 @@ class PlayerbotAI : public PlayerbotAIBase
     bool isPlayerFriend = false;
     bool isMovingToTransport = false;
     bool shouldLogOut = false;
+
+    QuestRewardOptionType m_questRewardOption;
```

Comment:
This should be stored in a value instead of on a class variable. See FocusHealTargetValue for an example (but save a integer/enum instead of a target). Ask me if you have any questions

Reply:
The requested changes have been implemented.
Tested By:
- local server environment, same functionality steps from pull request